NeuroID/src/androidLib/java/com/neuroid/tracker/callbacks/NIDActivityCallbacks.kt

```diff
@@ -99,17 +139,29 @@ class NIDActivityCallbacks: ActivityLifecycleCallbacks {
 
         if (existActivity) {
             if (hasFragments.not() && cameBackFromBehind.not()) {
-                registerTargetFromScreen(activity, registerTarget = true, registerListeners = false)
-            }
-            else {
+//                NIDLog.d("NID--Activity", "Activity - POST Started - Exist & fragments")
```

Comment:
we don't have to remove the logs lines since these are only going to debug right, unless these have no bearing anymore, then consider removing.